apps/frontend/src/libs/components/select/select.tsx

```diff
@@ -0,0 +1,81 @@
+import {
+	type ActionMeta,
+	type MultiValue,
+	type PropsValue,
+	type SingleValue,
+} from "react-select";
+import ReactSelect from "react-select";
+
+import { type Option } from "./libs/types/types.js";
+import styles from "./styles.module.css";
+
+type Properties<T = string> = {
+	isMulti?: boolean;
+	onChange: (
+		newValue: MultiValue<Option<T>> | SingleValue<Option<T>>,
+		actionMeta: ActionMeta<Option<T>>,
+	) => void;
+	options: Option<T>[];
+	placeholder?: string;
+	title?: string;
+	value: PropsValue<Option<T>>;
+};
+
+const Select = <T = string,>({
```

Comment:
you need to wrap it with react-form the same as our Input component.

Reply:
Oh, understood, thanks.